crml/nft/src/lib.rs

```diff
@@ -152,6 +164,34 @@ decl_error! {
 		AddToUniqueIssue,
 		/// Tokens with different individual royalties cannot be sold together
 		RoyaltiesProtection,
+		/// The mass_drop activation_time or limit is invalid
+		MassDropInvalid,
+		/// There is no mass_drop associated with the series
+		NoMassDropExists,
+		/// There are no accounts in the whitelist
+		WhitelistEmpty,
+		/// The account_id isn't in the whitelist
+		NoAccountId,
```

Comment:
could re-use `NoPermission` for this
```suggestion
		NoAccountId,
```